modules/installation-configuration-parameters.adoc

```diff
@@ -288,6 +288,10 @@ ifdef::aws[]
 |====
 |Parameter|Description|Values
 
+|`compute.platform.aws.amiID`
+|The AWS AMI used to boot compute machines for the cluster.
+|Any published or custom {op-system} AMI that belongs to the set AWS region.
+
```

Comment:
I noticed that `compute.platform.aws.amiID` , `controlPlane.platform.aws.amiID` and `platform.aws.amiID` are all under `Optional AWS parameters` table, but either `compute.platform.aws.amiID`, `controlPlane.platform.aws.amiID` or `platform.aws.amiID` is required for usgov region.

Reply:
@yunjiang29 the way the tables are set up currently, the **Required** params are really meant to reflect the params every AWS cluster must have set before deployment. The **Optional** params are made up of params that aren't required for all, but are certainly required in some cases (this is true for several others outside of GovCloud that are already listed).

I can treat this suggestion as an RFE, because I do see your point, but this would be a massive shift in the param listings that would affect all Installer sections (this is reused for every cloud platform installer doc).

What I'll do is add to the `amiID` param descriptions that they are required for AWS gov regions. That way it'll at least be clear it's required for certain cases, even though it's under *Optional AWS parameters*.

Reply:
@codyhoag I got your point, thanks, LGTM.